lib/rules/require-meta-schema-description.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+const utils = require('../utils');
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+/** @type {import('eslint').Rule.RuleModule} */
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description:
+        'require rules `meta.schema` properties to include descriptions',
+      category: 'Rules',
+      recommended: true,
```

Comment:
breaking change?

Reply:
yes, it follows the same policy as eslint: https://github.com/eslint/eslint?tab=readme-ov-file#semantic-versioning-policy.